### ISSUE TITLE
Feature request: Suggest the actions like the native LSP actions

### ISSUE THREAD

**Issue by lourenci** [ThePrimeagen/refactoring.nvim#108]
I don't use telescope, so this would be like a globe for me. Thank you.

<img width="944" alt="image" src="https://user-images.githubusercontent.com/2339362/138272680-426d00fc-3e44-4d5f-810d-a766feea15ca.png">

**Comment by adgai19** [comment]
If you are on nvim 6.0+ i think you could use this
```lua
local function ui_map()
	vim.ui.select(refactoring.get_refactors(), {
		prompt = "refactoring",
	}, function(refactor)
		refactor.refactor(refactor)
	end)
end

_G.Refactor_remaps = {
	ui_map = ui_map,
}
vim.api.nvim_set_keymap("v", "<leader>rf", "<cmd>lua Refactor_remaps.ui_map()<CR>", { silent = true })

```

**Comment by lourenci** [comment]
Yes. It's working. There is just a little issue when you cancel the options and don't make any selection:

<img width="2557" alt="image" src="https://user-images.githubusercontent.com/2339362/158706163-ae327aab-04b8-4c68-8698-1e147b983570.png">

**Comment by pranavrao145** [comment]
Alright, thanks for the feedback! I'll merge in a quick fix to this problem tomorrow and we'll close the issue.

**Comment by pranavrao145** [comment]
@lourenci I have merged in #273 to fix this - take a look and let me know what you think!

**Comment by pranavrao145** [comment]
Since it's been quite a while since #273 was merged, I'll close this issue for now. Please feel free to reopen if necessary.
